src/core.jl

```diff
@@ -201,16 +197,40 @@ every row in data.
 Data must have the same number of dimensions as the training dataset
 and will be normalised with the same parameters.
 """
-function mapToGigaSOM(som::Som, data)
-
-    data = convertTrainingData(data)
+function mapToGigaSOM(som::Som, data::DataFrame)
 
+    data::Array{Float64,2} = convertTrainingData(data)
     if size(data,2) != size(som.codes,2)
         println("    data: $(size(data,2)), codes: $(size(som.codes,2))")
         error(SOM_ERRORS[:ERR_COL_NUM])
     end
 
-    vis = visual(som.codes, data)
+    nWorkers = nprocs()
+    dData = distribute(data)
+    vis = Int64[]
+
+    if nWorkers > 1
+        # distribution across workers
+        R = Array{Future}(undef,nWorkers, 1)
+         @sync for p in workers()
+
+             println("worker: $p")
```

Comment:
Would it make sense to introduce here a `printLevel` variable?

Reply:
I'll take this out and only print the radius...